devices.js

```diff
@@ -167,17 +167,17 @@ const configureReporting = {
         const payload = [{
             attribute: 'occupiedHeatingSetpoint',
             minimumReportInterval: 0,
-            maximumReportInterval: repInterval.HOUR,
-            reportableChange: 0,
+            maximumReportInterval: repInterval.MINUTES_5,
```

Comment:
- Please change the max interval back to one hour, 5 minutes will probably drain the battery faster.
- Does `reportableChange: 1,` also work?

Reply:
* I increased the time to one hour for the temperature and the set point. I set `reportableChange: 10`. I think 1 (.01°C) is too low, temperature fluctuate it will report it quite often.
* This thermostat does not have battery because it is power from the line. 
* I kept short report cycle for the `pi_heating_demand` as I think it is  important for calculating the output power (and integrating for output energy).

Are you ok with that?